### ISSUE TITLE
Audit fixes

### ISSUE THREAD

**Issue by JodiNA** [SFDO-Community-Sprints/sprintys-community-resources#60]
In advance of the virtual sprint at the end of September, updates for auditing:
1) Make it so that curators can access audit history (check perms)
2) Change page layout for audit history to be helpful for logged in curators
3) Change "FAIL (Needs Improvements)" to "PASS (Needs Improvements)", as this is more intuitive. It seems like if it says fail, that the resource shouldn't be on the site.

**Comment by JodiNA** [comment]
@taniadoa did I forget anything on this that I said I'd do?

**Comment by taniadoa** [comment]
That was it! Thank you!

**Comment by JodiNA** [comment]
- I created a new page layout for curators and adjusted the Experience layout for the audit object.
- I updated curator profile permission to have read access to the audit object and added a sharing setting to give the Curator Public Group read access to all audit records.
 ![Image](https://github.com/user-attachments/assets/3ee2e2eb-036d-4432-8f3e-2a5c0ce7d1af)
- I added a new picklist value for audit status and updated the audit flow to use the new status. The flow uses a formula to set the audit status based on the information in the screen flow, so that's where the update was made. This formula is used to create the new audit record and on the "Last Audit Status" field on the Resource. I updated the existing resources and audit records to use the new status instead of the "FAIL" status. Deactivated the FAIL picklist value.
![Image](https://github.com/user-attachments/assets/63c00a5d-a75a-4ad8-8074-85f76e364dbe)
